src/main/java/service/WriteRecordsToDB.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import service.fruitrecord.FruitRecord;
+
+public interface WriteRecordsToDB {
+    void recordToMap(List<FruitRecord> fruitRecordList, ActivitiesStrategy strategy);
```

Comment:
Стратегію ми можемо передати через конструктор класу
```suggestion
    void recordToMap(List<FruitRecord> fruitRecords);
```